spring-boot-sample-project/src/main/java/com/example/demo/service/UserService.java

```diff
@@ -9,40 +10,28 @@
 
 @Component
 public class UserService {
-
-
+    @Autowired
+    private UserRRepository userRRepository;
     private UserRepository userRepository = new UserRepository();
 
     public List<User> getAllUser(){
-        List<User> userList = userRepository.getUserList();
+        List<User> userList = userRRepository.findAll();
         return userList;
     }
 
-    public User getAUser(String name){
-        List<User> userList = userRepository.getUserList();
-        for(User u : userList){
-            if(u.getFirstName().equalsIgnoreCase(name)){
-                return u;
-            }
-        }
-        return null;
+    public Optional<User> getAUser(int id){
+        Optional<User> u = userRRepository.findById(id);
+
+        return u;
     }
 
     public boolean createUser(User user){
-        userRepository.getUserList().add(user);
+        userRRepository.save(user);
```

Comment:
`POST` returns the created entity. in this case you should return the entity that got saved